src/components/common/view/CategoryBox.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { View } from 'react-native'
+import { theme } from '../../theme/styles'
+
+import ModalContents from '../modal/ModalContents'
+import ModalContainer from '../modal/ModalContainer'
+import { Section } from '../index'
+import { getShadowStyles } from '../../../lib/utils/getStyles'
+
+export const CategoryBox = ({ style, primarycolor, children, title, contentStyle }) => {
+  return (
```

Comment:
```suggestion
export const CategoryBox = ({ style, primarycolor, children, title, contentStyle }) => (
```

Reply:
for components without logic you could omit function body & return statement. Simply to put JSX just after the arrow

Reply:
done